code/exercises/affine-cipher/example.scm

```diff
@@ -0,0 +1,89 @@
+(import (rnrs))
+
+
+(define (encode key text)
+  ((get-translator key) (string-downcase text) 'encode))
+
+(define (decode key text)
+  ((get-translator key) text 'decode))
+
+;;; Private
+
+(define (atbash-format lst)
+  (let* ((chunks (windows lst 5))
+         (l (cons (car chunks)
+                  (map (lambda (chars) (cons #\space chars))
+                       (cdr chunks)))))
+    (list->string (fold-left append '() l))))
```

Comment:
Thought you would think this was fun.  It is more specific than I wanted, hence the specific name and placement above the generics